src/main/java/org/fcrepo/camel/FcrepoClient.java

```diff
@@ -81,7 +82,7 @@ public FcrepoResponse head(final URI url)
         final HttpRequestBase request = HttpMethods.HEAD.createRequest(url);
         final HttpResponse response = executeRequest(request);
         final int status = response.getStatusLine().getStatusCode();
-        final String contentType = getContentTypeHeader(response);
+        final Optional<String> maybeContentType = getContentTypeHeader(response);
```

Comment:
I am going to close this PR and view it as a practice exercise. I agree that it doesn't seem to add much in clarity.